compiler/src/test/scala/org/scalajs/nscplugin/test/OptimizationTest.scala

```diff
@@ -165,22 +165,6 @@ class OptimizationTest extends JSASTTest {
     }
   }
 
-  @Test
-  def testNewJSObjectAndJSArray: Unit = {
-    // Verify the optimized emitted code for 'new js.Object' and 'new js.Array'
-    """
-    import scala.scalajs.js
-
-    class A {
-      val o = new js.Object
-      val a = new js.Array
-    }
-    """.
-    hasNot("any reference to the global scope") {
-      case js.JSLinkingInfo() =>
```

Comment:
Oops. I think that test has been testing the wrong thing for ages.

Instead of deleting it, we should fix it to check that it does not contain any `js.JSGlobalRef` nor `js.LoadJSConstructor`.

Reply:
Oh, I see, I removed it because we anyway don't have `JSLinkingInfo` anyway, but yeah we should test it against `js.JSGlobalRef`s indeed 😄